File: api/core/src/main/kotlin/data/UcenecData.kt
package data

import domain.Audit
import domain.Naslov
import domain.Oseba
import kotlinx.serialization.Serializable

@Serializable
data class UcenecData(
    val oseba: Oseba,
    val naslov_refs: List<Naslov> = listOf(),
    val kontakt_refs: List<KontaktData> = listOf(),
    val ucenje_ucenec_refs: List<UcenjeData> = listOf(),
    val test_ucenec_refs: List<TestData> = listOf(),
    val audit_refs: List<Audit> = listOf()
)

File: app/src/app/services/api/openapi/models/ucenec-data.ts
/* tslint:disable */
/* eslint-disable */
import { AuditData } from './audit-data';
import { KontaktData } from './kontakt-data';
import { NaslovData } from './naslov-data';
import { Oseba } from './oseba';
import { TestData } from './test-data';
import { UcenjeData } from './ucenje-data';
export interface UcenecData {
  audit_refs?: Array<AuditData>;
  kontakt_refs?: Array<KontaktData>;
  naslov_refs?: Array<NaslovData>;
  oseba?: Oseba;
  test_ucenec_refs?: Array<TestData>;
  ucenje_ucenec_refs?: Array<UcenjeData>;
}
